lib/update.js

```diff
@@ -19,7 +25,9 @@ function resolveUpdates(updates, object) {
       let updatedValue = value
 
       if (
-        !Array.isArray(value) && value !== null && typeof value === 'object'
+        !Array.isArray(value) &&
```

Comment:
I'm down with prettiering the repo, but I'd prefer it be a separate commit/PR. Could you please amend any of the non-essential changes out?